Bottom: 49676af4878aeee1c7ee21695d88c65cfab69604
Top:    ae6277cc0ec1c676a1e4edcfefeb886293994282
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-03-20 19:55:09 +0800

Refresh of vhost_net-try-batch-dequing

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 12cdc8c..d14f2b3 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -1000,6 +1000,13 @@ static struct skb_array *get_tap_skb_array(int fd)
 	if (!file)
 		return NULL;
 	array = tun_get_skb_array(file);
+	if (!IS_ERR(array))
+		goto out;
+	array = tap_get_skb_array(file);
+	if (!IS_ERR(array))
+		goto out;
+	array = NULL;
+out:
 	fput(file);
 	return array;
 }
